purekit/src/main/kotlin/com/virgilsecurity/purekit/pure/PureCrypto.java

```diff
@@ -335,6 +364,7 @@ VirgilPublicKey importPublicKey(byte[] publicKey) throws PureCryptoException {
     byte[] encryptForBackup(byte[] plainText, VirgilPublicKey encryptKey, VirgilPrivateKey signingKey) throws PureCryptoException {
         try {
             return crypto.authEncrypt(plainText, signingKey, encryptKey);
+            //TODO CryptoException could be used here
```

Comment:
PureKit throws only subclasses of PureException

Reply:
If mean `catch (CryptoException e)`